packages/gatsby/src/commands/develop.ts

```diff
@@ -57,8 +57,11 @@ class ControllableScript {
   }
   start(): void {
     const args: Array<string> = []
+
+    const dotCachePath = path.join(process.cwd(), `.cache`)
+    fs.mkdirpSync(dotCachePath)
     const tmpFileName = tmp.tmpNameSync({
-      tmpdir: path.join(process.cwd(), `.cache`),
+      tmpdir: dotCachePath,
```

Comment:
https://github.com/raszi/node-tmp/commit/1cf4ec54180a77a2a95dc1941efa1659774c8787#diff-1cf97fc30a5d84b74b60007e190ead611730d702c2dd9d353586dfa16862bb5dR29-R486 this change that was recently released in `tmp` package make it so provided `tmpdir` need to exist, while previous patch version of package didn't need that

so instead of pinning, I just ensure dir exist